coralproject/talk
# CSS and Template Fixes + Improvements

## What does this PR do?

- [Fixed CSS ordering bug](https://www.pivotaltracker.com/story/show/157434681)
- Improved developer experience

## How do I test this PR?

1. Add custom CSS rules to overrride a property in the embed stream, taking care to **not** use `!important`.
2. Refresh the embed stream to see the new design with the change applied.
